### ISSUE TITLE
Paths (baseUrl) with Nodejitsu (Joyent/SmartOS)

### ISSUE THREAD

**Issue by azampagl** [requirejs/requirejs#541]
Hi,

   TL;DR: SmartOS (Nodejitsu/Joyent) does not seem to properly recognize relative file locations (absolute paths are always required).

**Does Not Work**:

```
requirejs(['myModule'], function (myModule) { ... });
```

**Works**:

```
requirejs(['/some/absolute/path/myModule.js'], function (myModule) { ... });
```

   I'm trying to use requirejs (v2.1.1) with node (v0.8.x) an am running into path problems using Nodejitsu (Joyent/SmartOS).  Essentially, I'm trying include a module (./app/app.js) from my bootstrap script (./server.js).

   https://gist.github.com/4137715

   However, the operating system does not recognize relative paths.  I continuously get this error:

   https://gist.github.com/4092457 

   The response I got back from Nodejitsu support was 

> It is still looking for a node module and not a local file, This sometime is different than other systems as SmartOS file sytem has some oddities.
> 
> try adding './' then the filenames 

   Unfortunately, not event their suggestion worked.  I had to use the absolute path in order for requirejs to get load the module:

   https://gist.github.com/4137739

   Any suggestions :)?

Thanks for your time,
Aaron
